app/src/interfaces/file-image/file-image.vue

```diff
@@ -117,15 +125,31 @@ export default defineComponent({
 				return addTokenToURL(getRootPath() + `assets/${image.value.id}`);
 			}
 			if (image.value.type.includes('image')) {
-				const fit = props.crop ? 'cover' : 'contain';
+				const fitParam = props.crop ? 'cover' : 'inside';
 				const url =
-					getRootPath() + `assets/${image.value.id}?key=system-large-${fit}&cache-buster=${cacheBuster.value}`;
+					getRootPath() +
+					`assets/${image.value.id}?width=800&height=800&fit=${fitParam}&withoutEnlargement=true&cache-buster=${cacheBuster.value}`;
```

Comment:
We'll have to keep using the system included `key` parameters here. The user can disable the feature to use the other query parameters from Settings, so we gotta make sure that doesn't break the file library completely